frontend/src/app/libs/container/components/localization/localization.component.scss

```diff
@@ -5,14 +5,20 @@
   @include drop-down-menu-container;
 
   &__menu-item {
-		@include menu-item-hover;
-		
-    text-align: center;
-    color: $color-white;
+	@include menu-item-hover;
+	color: $color-white;
+	width:152px;
+	margin:8px 0;
   }
-
   &__menu-trigger {
     display: flex;
-    align-items: center;
+	align-items: center;
+  }
+  &__language {
+	  @include text-p();
   }
+   &__flag {
```

Comment:
same here, please check indendation (tabs, 4 spaces), and add new empty line between selectors